PokeAlarm/Alarms/Telegram/TelegramAlarm.py

```diff
@@ -24,54 +24,63 @@ class TelegramAlarm(Alarm):
 
     Alert = namedtuple(
         "Alert", ['bot_token', 'chat_id', 'sticker', 'sticker_url',
-                  'sticker_notify', 'message', 'message_notify', 'venue',
+                  'sticker_notify', 'message', 'venue_message',
+                  'message_notify', 'venue',
                   'venue_notify', 'map', 'map_notify', 'max_attempts',
                   'web_preview'])
 
     _defaults = {  # No touchy!!! Edit alarms.json!
         'monsters': {
             'message': "*A wild <mon_name> has appeared!*\n"
                        "Available until <24h_time> (<time_left>).",
+            'venue_message': "",
```

Comment:
Wouldn't the proper thing here be to set `venue_message` to `null` rather than an empty string?